src/main/java/Duke.java

```diff
@@ -1,10 +1,184 @@
+import java.util.Scanner;
+
+
 public class Duke {
+    private static Task[] taskList = new Task[100];
+    private static String[] storeUserText = new String[100];
+    private static boolean doExit = false;
+
     public static void main(String[] args) {
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
+        lineSeparator();
+        greet();
+        lineSeparator();
+
+        while (!doExit) {
+            String userInput = getInput();
+            try {
+
+                String[] userInputArray = checkInput(userInput);
+
+            //firstWord will decide the actions that will be done
+            String firstWord = userInputArray[0];
+
+            if (firstWord.equals("bye")) {
```

Comment:
Everything inside a try...catch block should be indented. They cannot be on the same level of indentation as "try" itself.